Modules/Microsoft365DSC/DSCResources/MSFT_SCAutoSensitivityLabelRule/MSFT_SCAutoSensitivityLabelRule.psm1

```diff
@@ -1275,81 +1275,84 @@ function ConvertTo-SCDLPSensitiveInformationStringGroup
 
     foreach ($SensitiveInformationHash in $InformationArray)
     {
-        $StringContent = "MSFT_SCDLPContainsSensitiveInformation`r`n            {`r`n"
+        $StringContent = "MSFT_SCDLPContainsSensitiveInformation {`r`n"
```

Comment:
Please keep the opening bracket '{' next to the CIM instance name without spaces like 'MSFT_SCDLPContainsSensitiveInformation{', our application relies on ConvertTo-DSCObject from DSCParser to get the info inside blueprints and from my experience any spaces or carriage returns after the instance name causes several problems converting the resources. The obvious solution here would be to instead permit any number of white spaces between the name and the bracket in DSCParser but alas...

Same applies to remaining code below, thank you for your comprehension.

Reply:
This update is to fix exactly that. I am working with a customer that has issues with parsing the current exported resource into an object and have tested if these fixes correct the issue, which it did. Even with the open brace on the same line.

Reply:
Yeah just tested it and ConvertTo-DSCObject seems to not only accept the brace with spaces it also accepts them on a new line now, Nik must have fixed it in some update and I never bother to check that again.

Thank you for the fix on this resource.